src/kili/entrypoints/mutations/asset/__init__.py

```diff
@@ -37,13 +36,13 @@ class MutationsAsset:
 
     # pylint: disable=too-many-arguments,too-many-locals
 
-    def __init__(self, auth: KiliAuth):
+    def __init__(self, kili) -> None:
         """Initialize the subclass.
 
         Args:
-            auth: KiliAuth object
+            kili: Kili client.
         """
-        self.auth = auth
+        self.kili = kili
```

Comment:
wasn't it possible to only give the graphql_client here ?
the Queries and Mutations class get the Kili client as input and the Kili client extends these class. Isn't it dangerous ?